src/detect-flowbits.c

```diff
@@ -69,6 +72,39 @@ void DetectFlowbitsRegister (void)
     DetectSetupParseRegexes(PARSE_REGEX, &parse_regex);
 }
 
+static int FlowbitOrAddData(char *arrptr, DetectFlowbitsData *cd, DetectEngineCtx *de_ctx)
```

Comment:
nit: convention in other code is to have `de_ctx` as the first arg, the `cd` following that and the more specific args(s) last.